lib/SIL/SILPrinter.cpp

```diff
@@ -3026,6 +3057,72 @@ void SILDefaultWitnessTable::dump() const {
   print(llvm::errs());
 }
 
+// SWIFT_ENABLE_TENSORFLOW
+void SILDifferentiabilityWitness::print(
+    llvm::raw_ostream &OS, bool verbose) const {
+  OS << "// differentiability witness for "
+     << demangleSymbol(originalFunction->getName()) << "\n";
+  // sil_differentiability_witness @original-function-name : $original-sil-type
```

Comment:
Correct the syntax.
```suggestion
  // sil_differentiability_witness [parameters ...] [results ...] [where ...] @original-function-name : $original-sil-type
```

Reply:
The suggested comment is a bit too long. Currently, each printed section has its own comment, which I think is a bit clearer, if that's okay.

Clarified comments here to be a bit more clear.